README.md

```diff
@@ -69,6 +85,6 @@ Exported as: `knowledgeOwlLink`
 
 ## Properties
 
-| Name                                     | Required | Description              |
-| ---------------------------------------- | -------- | ------------------------ |
-| @Input() <br/>`knowledgeOwlLink: string` | true     | KnowledgeOwl Article URL |
+| Name                                     | Required | Description               |
+| ---------------------------------------- | -------- | ------------------------- |
+| @Input() <br/>`knowledgeOwlLink: string` | true     | KnowledgeOwl Article Name |
```

Comment:
I think it's the complete article URL, not the name. Isn't it? @somanwal

Reply:
@so-dinesh-rawat This is just the name of Article. URL will be constructed automatically with provider.